grpc/src/test/java/com/linecorp/armeria/server/grpc/AsyncServerInterceptorTest.java

```diff
@@ -87,6 +91,12 @@ void authorizedRequest(String path) {
                                                                       .setFillUsername(true)
                                                                       .build());
         assertThat(response.getUsername()).isEqualTo("Armeria");
+        assertEquals(0, exceptionCounter.get());
```

Comment:
Code style: We prefer to use `assertThat` of AssertJ

Reply:
Fixed